__tests__/helpers.js

```diff
@@ -0,0 +1,47 @@
+"use strict";
+
+const MemoryFileSystem = require("memory-fs");
+const webpack = require("webpack");
+
+exports.PluginEnvironment = class PluginEnvironment {
```

Comment:
Would it also be possible to get the `MemoryFS` output 'bundle' and snapshot test it? 

```js
test('${name}', () => {
   compiler(fixture, config)
     .then((result) => {
        result.stats
        result.output // MemoryFS Instance as `bundle.js`
      })
     .catch((err) => err)
})